src/app/tweets/tweets.component.html

```diff
@@ -45,35 +50,50 @@ <h4 class="header-title m-t-0">All Tweets</h4>
       <p class="text-muted m-b-25 font-13">
         All tweets from @realDonaldTrump
       </p>
-      <table class="table mb-0 table-responsive">
-        <thead>
-          <tr>
-            <th>Date</th>
-            <th>Time</th>
-            <th>Phrases</th>
-            <th>Sentiment</th>
-            <th>Tweet</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr *ngFor="let tweet of tweets$ | async | slice:0:100 ">
-            <td>{{tweet.ts | date: 'fullDate'}}</td>
-            <td>{{tweet.ts | date: 'shortTime'}}</td>
-            <td>
-              <div *ngFor="let phrase of tweet.phrases | slice:0:3">
-                <ins class="text-default">
-                  {{phrase}}</ins>&nbsp; |
-              </div>
-            </td>
-            <td>
-              <img src="assets/images/logo_postive.png" width="80" height="80" alt="logo" *ngIf="tweet.sentiment > 0.66">
-              <img src="assets/images/logo.png" width="80" height="80" alt="logo" *ngIf="tweet.sentiment <= 0.66 && tweet.sentiment >= 0.33">
-              <img src="assets/images/logo_negative.png" width="80" height="80" alt="logo" *ngIf="tweet.sentiment < 0.33">
-            </td>
-            <td>{{tweet.text}}</td>
-          </tr>
-        </tbody>
-      </table>
+
+      <div class="list-group">
+        <a href="#"
+           *ngFor="let tweet of tweets$ | async | slice:0:100"
+           class="list-group-item list-group-item-action flex-column align-items-start">
+          <div class="d-flex w-100 justify-content-between">
+            <h5 class="mb-2">
+
+              <img src="assets/images/logo_postive.png"
```

Comment:
Having a look at the end result when trying to make the face smaller you really can't tell if it's  red or green :(

So after second thoughts can we could turn this into some emojis instead? Maybe 👎  for negative 👍 for positive and 🤷  for unsure.

Reply:
I've made this change.  Other combinations that might work are 🙂☹😕, or maybe just switching 🤷to 😕.  I didn't immediately recognize this as uncertainty, but that doesn't mean anyone else would have that problem.